executor/src/witgen/generator.rs

```diff
@@ -485,11 +504,17 @@ where
                     assert!(!progress);
                 }
                 for (id, c) in constraints.constraints {
+                    if first {
+                        log::trace!("    Processing: {}", source_name());
```

Comment:
Maybe we should just always print the identity outside of this function? I.e. every single identity we are processing? Then we also know the order in which they are processed.

Reply:
That was my first attempt, but that got very verbose, because it'll list all identities multiple times for each row.